mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/MavenPropertyResolver.java

```diff
@@ -18,4 +24,14 @@ static private boolean isSkip() {
         return Boolean.valueOf(System.getProperty("maven.test.skip", "false"));
     }
 
+    static boolean isSpecificTestClassSet() {
+        String testClasses = System.getProperty("test");
+        return testClasses != null && !containsPattern(testClasses);
+    }
+
+    private static boolean containsPattern(String testClasses) {
+        return Arrays.stream(testClasses.split(","))
+            .map(TEST_CLASS_PATTERN::matcher)
+            .anyMatch(Matcher::find);
+    }
```

Comment:
Yes @bartoszmajsak, I have added the tests and renamed it to `testClasses` as `specificTestClasses` was slightly confusing.